Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -301,5 +309,157 @@ configuring the CiliumLocalRedirectPolicy.
+           ``hostNetwork: false`` for the daemonset.
+
+         * In the Corefile, bind to ``0.0.0.0`` instead of the static IP.
+
+         * In the Corefile, let CoreDNS serve health-check on its own IP instead of the static IP by
+           removing the host IP string after health plugin.
+
+         * Modify Node-local DNS cache's deployment yaml to point readiness probe to its own IP by
+           removing the ``host`` field under ``readinessProbe``.
+
+* Deploy local redirect policy (LRP) to steer DNS traffic to the node local dns cache.
+
+  .. parsed-literal::
+
+      $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/node-local-dns-lrp.yaml
+
+  .. note::
+
+      * The LRP above uses ``kube-dns`` for the cluster DNS service, however if your cluster DNS service is different,
+        you will need to modify this example LRP to specify it.
+      * The namespace specified in the LRP above is set to the same namespace
+        as the node-local DNS cache daemonset.
+      * The LRP above uses the same port names ``dns`` and ``dns-tcp`` as the example quick deployment yaml, you will
+        need to modify those to match your deployment if they are different.
+
+After all ``node-local-dns`` pods are in ready status, DNS traffic will now go to the local node-cache first.
+You can verify by checking the DNS cache's metrics such as ``coredns_cache_hits_total`` and
+``coredns_cache_misses_total`` via curling ``<node-local-dns-pod>:9253/metrics``. A new DNS request either
+hits or misses in the cache, therefore incrementing the corresponding metric.
+
+kiam redirect on EKS
+--------------------
+`kiam <https://github.com/uswitch/kiam>`_ agent runs on each node in an EKS
+cluster, and intercepts requests going to the AWS metadata server to fetch
+security credentials for pods.
+
+- In order to only redirect traffic from pods to the kiam agent, and pass
+  traffic from the kiam agent to the AWS metadata server without any redirection,
+  we need the socket lookup functionality in the datapath. This functionality
+  requires v5.1.16, v5.2.0 or more recent Linux kernel. Make sure the kernel
+  version installed on EKS cluster nodes satisfies these requirements.
+
+- Deploy `kiam <https://github.com/uswitch/kiam>`_ using helm charts.
+
+  .. code-block:: bash
+
+      $ helm repo add uswitch https://uswitch.github.io/kiam-helm-charts/charts/
+      $ helm repo update
+      $ helm template kiam uswitch/kiam > kiam.yaml
+
+  - If you see an error like "request blocked by whitelist-route-regexp" while
+    running requests to the metadata server, then you may need to whitelist the
+    metadata requests by passing the below argument to the ``kiam-agent Deamonset``.
+
+    .. code-block:: bash
+
+        $ sed -i '/args:/a \ \ \ \ \ \ \ \ \ \ \ \ - --whitelist-route-regexp=meta-data' kiam.yaml
+
+
+  - Make sure the "--iptables" argument is removed from the arguments passed
+    to the ``kiam-agent Daemonset``.
```

Comment:
Were you planning to update the `helm template` command above to disable this via helm to avoid the manual instruction here?

Reply:
The default _is_ false.